Tiber upgrade (#259)
autonity/docs.autonity.org: reference/api/liquid-newton/index.md

You will need to specify the validator's Liquid Newton contract address. This can be retrieved using the `aut` command `aut validator info` - for usage see Autonity Contract Interface and [`getValidator()`](/reference/api/aut/#getvalidator).
:::

## claimRewards

Computes the total rewards owed to the caller, and sends the appropriate amount of auton.

The `aut validator claim-rewards` command uses this function.  Usage details and examples are given in the ["Claiming staking rewards"](/delegators/claim-rewards/#claim-rewards) section.

## claimTreasuryATN

> Computes the total rewards owed to the validator treasury from commission charged on [delegated](/glossary/#delegated) stake, and sends the appropriate amount of auton to the validator's  `treasury` account.

Reviewer: Reward computation doesn't happen in this function, but rather when the AC calls `redistribute`.
```suggestion
Sends to the treasury the ATN rewards accrued by the validator via the commission charged on [delegated](/glossary/#delegated) stake.
```

Constraint checks are applied:

- the caller is the validator `treasury` address.

### Parameters
   
None.

### Response

None.

### Event

None.

### Usage

::: {.callout-note title="Note" collapse="false"}
The `claimTreasuryATN()` function in the Liquid Newton Contract Interface is not currently supported by `aut`.

You can interact with the contract using the `aut contract` command group. See `aut contract tx -h` for how to submit a transaction calling the interface function.
:::

## getTreasuryUnclaimedATN

Queries the contract and returns the total unclaimed ATN rewards earned by a validator from commission charged on [delegated](/glossary/#delegated) stake.

> The rewards are claimed and sent to the validator's `treasury` by the validator operator calling [`claimTreasuryATN()`](/reference/api/liquid-newton/#claimtreasuryatn).

Reviewer: ```suggestion
The rewards can be claimed and sent to the validator's `treasury` by the validator operator calling [`claimTreasuryATN()`](/reference/api/liquid-newton/#claimtreasuryatn).
```

Author: That changes it from the active to the passive voice, and it's best practice to use passive:

https://developers.google.com/tech-writing/one/active-voice

Resolving.

Reviewer: @cmjc  my point is that using `are` suggests that no action is required from the user. While instead claiming rewards is something the user needs to actively do.

0
```
:::

## unclaimedRewards

> Queries the contract and returns the total rewards owed to a given account.

Reviewer: ```suggestion
Queries the contract and returns the total ATN rewards owed to a given account.
```

> Queries the contract and returns the total ATN rewards owed to a given account.

Reviewer 2: Not sure what is served by saying "queries the contract":

```suggestion
Returns the total ATN rewards owed to a given account.
```
